### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ostUp**

### CODE REVIEW

**Review by oksana-logos-frontend — APPROVED** [review]
Good Job!🔥

File: src/convertToObject.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const resultArray = sourceString.split(';').map(el => el.trim())
+    .filter(el => el.length > 0)
+    .map(el => el.split(':'))
+    .map(el => el.map(word => word.trim()));
```
Comment (AndriiRut) [inline src/convertToObject.js:9]: Good, but use better naming,  to increase code readability.
checklist # 2

**Review by AndriiRut — CHANGES_REQUESTED** [review]

**Review by AndriiRut — APPROVED** [review]
🚀

**Review by Bohdan-Maksymiuk — APPROVED** [review]
Great job 👏

**Pull request closed without merging**